back/src/modules/products/model/products.model.ts

```diff
@@ -38,14 +38,6 @@ export class Product {
   @Prop()
   description: string;
 
-  @Field(() => Float)
-  @Prop()
-  price: number;
-
-  @Field(() => Int)
-  @Prop()
-  percentDiscount: number;
-
   @Field(() => [Category], { nullable: true })
   @Prop({ type: MongooseTypes.ObjectId, ref: 'Category' })
   categories: Category[];
```

Comment:
**Suggestion:** Similar to the previous suggestion, it seems like you removed the price and percentDiscount fields from the Product model. If these fields are no longer needed, this is fine. However, if you still need these fields, you should add them back.
```suggestion
  @Field(() => Float)
  @Prop()
  price: number;

  @Field(() => Int)
  @Prop()
  percentDiscount: number;

  @Field(() => [Category], { nullable: true })
  @Prop({ type: MongooseTypes.ObjectId, ref: 'Category' })
  categories: Category[];
```